home_work_3/task2.py

```diff
@@ -0,0 +1,16 @@
+#  Напишите программу, которая найдёт произведение пар чисел списка.
+#  Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+#  Пример:
+#  [2, 3, 4, 5, 6] => [12, 15, 16];
+#  [2, 3, 5, 6] => [12, 15]
+
+import random
+zol = [random.randint(0, 10) for _ in range(4)]
+print(zol)
+zol2=[]
+m = 1
+for i in range((len(zol) + 1) // 2):
+    m = zol[i] * zol[- i - 1]
```

Comment:
```suggestion
    m = zol[i] * zol[-i - 1]
```
Это унарный минус, его не нужно писать через пробел.